src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/EgressExtensibilityTests.cs

```diff
@@ -132,6 +133,33 @@ private IEgressExtension FindEgressExtension(ConfigDirectory configDirectory, Ho
 
             CopyExtensionFiles(extensionDirPath);
 
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Would it be possible to shift this into the test pipeline itself instead of doing it in the code?

Reply:
I don't think so. Each invocation of the test creates a new temporary directory to which this is copied. The pipeline won't know what that directory is.

We can take a look at running the extension as-is from where it was built/downloaded rather than copying it to a new directory, unless the point of the test is to test that it can run elsewhere.